src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,38 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+    let str = "";
+    let fullStr = "";
+    let count = 0;
+    array2 = [];
+    if (!(typeof text === 'string')){
+        throw("Please enter a valid string");
```

Comment:
Prefer to `throw Error(...)` rather than strings. Helps preserve the stack trace for debugging.